Distancify.Migrations.Litium/Seeds/Products/PriceListItemSeed.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Linq;
+using Litium;
+using Litium.Products;
+
+namespace Distancify.Migrations.Litium.Seeds.Products
+{
+    public class PriceListItemSeed : ISeed
+    {
+        private PriceListItem priceListItem;
+
+        public PriceListItemSeed(PriceListItem inventoryItem)
+        {
+            this.priceListItem = inventoryItem;
+        }
+
+        public Guid Commit()
+        {
+            var service = IoC.Resolve<PriceListItemService>();
+
+            if (priceListItem.SystemId == null || priceListItem.SystemId == Guid.Empty)
+            {
+                priceListItem.SystemId = Guid.NewGuid();
+                service.Create(priceListItem);
+            }
+            else
+            {
+                service.Update(priceListItem);
+            }
+
+            return priceListItem.SystemId;
+        }
+
+        public static PriceListItemSeed Ensure(string priceListId, string variantId)
+        {
+            return Ensure(priceListId, variantId, default);
+        }
+
+        public static PriceListItemSeed Ensure(string priceListId, string variantId, decimal minimumQuantity)
+        {
+            var priceListSystemId = IoC.Resolve<PriceListService>().Get(priceListId).SystemId;
+            var variantSystemId = IoC.Resolve<VariantService>().Get(variantId).SystemId;
+            
+            var priceListItem = IoC.Resolve<PriceListItemService>().Get(variantSystemId, priceListSystemId)
+                .FirstOrDefault(i=>i.MinimumQuantity == minimumQuantity)?.MakeWritableClone() ??
```

Comment:
This is one huge null-coalescing operator. An if-statement would be more readable.

Reply:
I fixed this in the latest pull